listenbrainz/webserver/templates/footer.html

```diff
@@ -0,0 +1,323 @@
+                                target="_blank"
+                        >
+                            Shop
+                        </a>
+                    </li>
+                    <li>
+                        <a
+                                class="fw-bold "
+                                href="https://metabrainz.org/contact"
+                                rel="noopener noreferrer"
+                                target="_blank"
+                        >
+                            Contact us
+                        </a>
+                    </li>
+                </ul>
+            </div>
+        </div>
+        <div class="row mt-4 ms-3 me-3">
+            <div class="col-md-3 border-top pt-4 d-none d-md-block">
+                <p>
+                    OSS Geek?
+                    <a
+                            href="https://github.com/metabrainz/musicbrainz-server"
+                            rel="noopener noreferrer"
+                            target="_blank"
+                    >
+                <span>
+                  Contribute here
+                </span>
+                    </a>
+                </p>
+            </div>
+            <div class="col-md-6 border-top pt-4 text-center">
+                Brought to you by <img src="{{ url_for('static', filename='img/meb-icons/MetaBrainz.svg') }}" width="30" height="30" alt="MetaBrainz"/> <span> MetaBrainz Foundation </span>
+            </div>
+            <div class="col-md-3 border-top pt-4 d-none d-md-block">
+                <p>
+                    Found an Issue?
+                    <a
+                            class="ms-1"
+                            href="https://tickets.metabrainz.org/"
+                            rel="noopener noreferrer"
+                            target="_blank"
+                    >
+                <span>
+                  Report it here
+                </span>
+                    </a>
+                </p>
+            </div>
+        </div>
+    </div>
+</section>
+
+
+<!--<section class="footer">-->
+<!--    <div class="container-fluid">-->
+<!--        <div class="row">-->
+<!--            <div class="col-sm-12 col-md-6">-->
+<!--                <h3>-->
+<!--                    <img src="{{ url_for('static', filename='img/listenbrainz-logo.svg') }}" width="180" alt="ListenBrainz"/>-->
+<!--                </h3>-->
+<!--                <br/>-->
+<!--                <p class="color-gray">-->
+<!--                    ListenBrainz keeps tracks of what music you listen to and provides you with insights into your listening habits.-->
+<!--                    <br>-->
+<!--                    You can use ListenBrainz to track your listening habits, discover new music with personalized recommendations, and share your musical taste with others using our visualizations.-->
+<!--                </p>-->
+<!--                <ul class="list-unstyled">-->
+<!--                    <li class="color-a">-->
```

Comment:
I suppose all this commented code should be removed?

Reply:
Right!